Mathlib/Analysis/RCLike/Basic.lean

```diff
@@ -51,6 +48,7 @@ local notation "𝓚" => algebraMap ℝ _
 
 open ComplexConjugate
 
+set_option autoImplicit true in
```

Comment:
Why is this one necessary?

Reply:
It wasn't; thanks for prodding me to remove it!